docs/excel/performance.md

```diff
@@ -116,6 +120,18 @@ Excel.run(async function(ctx) {
 })
 ```
 
+### Suspend screen updating
+
+> [!NOTE]
+> The `suspendScreenUpdatingUntilNextSync()` method described in this article requires the beta version of the Office JavaScript library from [Office.js CDN](https://appsforoffice.microsoft.com/lib/beta/hosted/office.js). The [type definition file] (https://appsforoffice.microsoft.com/lib/beta/hosted/office.d.ts) is also found at the CDN. For more information on our upcoming APIs, please visit the [open spec](https://github.com/OfficeDev/office-js-docs/tree/ExcelJs_OpenSpec) on GitHub.
+
+Excel displays changes your add-in makes approximately as they happen in the code. For large, iterative data sets, you may not need to see this progress on the screen. `Application.suspendScreenUpdatingUntilNextSync()` pauses visual updates to Excel until the ad-in calls `context.sync()` (or the `Excel.run` call ends, implictly calling `context.sync`). Be aware, Excel will not show any signs of activity until the next sync. Your add-in should either give users guidance to prepare them for this delay or provide a status bar to demonstrate activity.
```

Comment:
"...pauses visual updates to Excel until the ad-in calls `context.sync()` (or the `Excel.run` call ends, implictly calling `context.sync`).
->
"...pauses visual updates to Excel until the add-in calls `context.sync()`, or until the `Excel.run` ends (implicitly calling `context.sync`)."